src/gc-arena/tests/ui/bad_collect_bound.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0277]: the trait bound `NotCollect: Collect` is not satisfied
+  --> $DIR/bad_collect_bound.rs:8:5
+   |
+8  |     field: NotCollect
+   |     ^^^^^^^^^^^^^^^^^ the trait `Collect` is not implemented for `NotCollect`
+   |
+  ::: $WORKSPACE/src/gc-arena/src/collect.rs
+   |
+   |         Self: Sized,
+   |               ----- required by this bound in `needs_trace`
```

Comment:
This is due to https://github.com/rust-lang/rust/issues/82038 - once that issue is fixed, this extra line should go away.